rpcserver.go

```diff
@@ -430,6 +430,10 @@ func mainRPCServerPermissions() map[string][]bakery.Op {
 			Entity: "macaroon",
 			Action: "generate",
 		}},
+		"/lnrpc.Lightning/ListPermissions": {{
```

Comment:
👍 

In the future since these macaroon management calls have started to grow a bit, we could consider moving them into their own sub-server.

Reply:
Yes, definitely. Though I'm not sure how we'd keep the backward compatibility when moving them, as they'd get a new URI (for example `/macaroonrpc.Macaroons/ListPermissions`) and break for users with an old proto. Or would we just deprecate them for a whole release on the main RPC and eventually remove them there?